src/frontend/Pages/Home/Home.css

```diff
@@ -0,0 +1,152 @@
+html {
+
+    height: 100%;
+    --cta-light: #fbcfe8;
+    --border: #e5f0f0;
+}
+
+.hero-sec {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 50vh;
+    padding: 0;
+    background-image: linear-gradient(to right, #ff1b6b, #45caff);
+
+}
+
+.hero-txt {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
Suggestion : You can make utility class for these property as it will use at many places.

Reply:
Thak you gopal i have made context for data